packages/api-explorer/src/scenes/MethodScene/MethodScene.tsx

```diff
@@ -72,10 +74,10 @@ export const MethodScene: FC<DocMethodProps> = ({ api }) => {
       <DocMarkdown source={method.description} specKey={specKey} />
       <DocSDKs api={api} method={method} />
       {seeTypes.length > 0 && (
-        <>
+        <Box mb="large">
           Referenced types:
           <DocReferences items={seeTypes} api={api} specKey={specKey} />
-        </>
+        </Box>
```

Comment:
`Box` needs to be imported. Currently this breaks during compilation.

Reply:
Not using Box here anymore.